doc/guide/build.hpp

```diff
@@ -104,6 +105,17 @@ In Ubuntu and Debian, you can get all of these dependencies through apt:
   libstb-dev
 @endcode
 
+@note For older versions of Ubuntu, Armadillo needs to be built from 
+source as apt installs an older version. So you need to omit 
+\c libarmadillo-dev from the code snippet above. Then 
+<a href="http://arma.sourceforge.net/download.html">this link</a>
+can be used to download the file and the README in the uncompressed folder
```

Comment:
I'd reword this just a little. Maybe something like "Download and extract the <a href="http://arma.sourceforge.net/download.html">armadillo source</a>, and follow the instructions to build and install it."

Reply:
Sure I'll do that.